sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.institutions.controller.getAssessmentNOA.e2e-spec.ts

```diff
@@ -65,21 +67,147 @@ describe("AssessmentInstitutionsController(e2e)-getAssessmentNOA", () => {
+      assessment.disbursementSchedules;
 
-    const awards = {};
-    firstDisbursementSchedule.disbursementValues.forEach((disbursement) => {
-      awards[`disbursement1${disbursement.valueCode.toLowerCase()}`] =
-        disbursement.valueAmount;
-    });
+    const firstDisbursementScheduleAwards = createDisbursementScheduleAwards(
+      firstDisbursementSchedule,
+      1,
+    );
+
+    const secondDisbursementScheduleAwards = createDisbursementScheduleAwards(
+      secondDisbursementSchedule,
+      2,
+    );
+
+    const endpoint = `/institutions/assessment/student/${student.id}/application/${application.id}/assessment/${assessment.id}/noa`;
+    const institutionUserToken = await getInstitutionToken(
+      InstitutionTokenTypes.CollegeFUser,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .expect({
+        applicationId: application.id,
+        applicationNumber: application.applicationNumber,
+        applicationStatus: application.applicationStatus,
+        assessment: assessment.assessmentData,
+        disbursement: {
+          disbursement1COEStatus: firstDisbursementSchedule.coeStatus,
+          disbursement1Date: getDateOnlyFormat(
+            firstDisbursementSchedule.disbursementDate,
+          ),
+          disbursement1Id: firstDisbursementSchedule.id,
+          disbursement1MSFAACancelledDate:
+            firstDisbursementSchedule.msfaaNumber?.cancelledDate,
+          disbursement1MSFAADateSigned:
+            firstDisbursementSchedule.msfaaNumber?.dateSigned,
+          disbursement1MSFAAId: firstDisbursementSchedule.msfaaNumber?.id,
+          disbursement1MSFAANumber: MASKED_MSFAA_NUMBER,
+          disbursement1Status:
+            firstDisbursementSchedule.disbursementScheduleStatus,
+          disbursement1TuitionRemittance:
+            firstDisbursementSchedule.tuitionRemittanceRequestedAmount,
+          ...firstDisbursementScheduleAwards,
+          disbursement2COEStatus: secondDisbursementSchedule.coeStatus,
+          disbursement2Date: getDateOnlyFormat(
+            secondDisbursementSchedule.disbursementDate,
+          ),
+          disbursement2Id: secondDisbursementSchedule.id,
+          disbursement2MSFAACancelledDate:
+            secondDisbursementSchedule.msfaaNumber?.cancelledDate,
+          disbursement2MSFAADateSigned:
+            secondDisbursementSchedule.msfaaNumber?.dateSigned,
+          disbursement2MSFAAId: secondDisbursementSchedule.msfaaNumber?.id,
+          disbursement2MSFAANumber: MASKED_MSFAA_NUMBER,
+          disbursement2Status:
+            secondDisbursementSchedule.disbursementScheduleStatus,
+          disbursement2TuitionRemittance:
+            secondDisbursementSchedule.tuitionRemittanceRequestedAmount,
+          ...secondDisbursementScheduleAwards,
+        },
+        eligibleAmount: assessment.disbursementSchedules.reduce(
```

Comment:
I have two points here.

First, when doing this type of E2E test we can also consider the option to control the values saved to the DB (the factories allow it) and then do the expected check as, for instance, 
```ts
eligibleAmount: 984321 // Sum of all controlled values manually calculated based on the DB values manually assigned.
```

Second, if we are not controlling the values as suggested on the "First" point (which I would prefer) I would consider creating a local method similar to the `sumDisbursementValueAmounts`.